src/main/java/org/sagebionetworks/web/client/widget/entity/browse/EntityTreeBrowserViewImpl.java

```diff
@@ -86,9 +77,10 @@ public void showErrorMessage(String message) {
 
 	@Override
 	public void showLoading() {
-		TreeItem loadingTreeItem = new TreeItem(new HTMLPanel(DisplayUtils.getLoadingHtml(sageImageBundle)));
```

Comment:
(to support loading child elements).  also change More button state to indicate loading.